alibi/explainers/integrated_gradients.py

```diff
@@ -798,11 +867,16 @@ def explain(self,
             If not provided, all features values for the baselines are set to 0.
         target
             Defines which element of the model output is considered to compute the gradients.
-            It can be a list of integers or a numeric value. If a numeric value is passed, the gradients are calculated
-            for the same element of the output for all data points.
-            It must be provided if the model output dimension is higher than 1.
+            Target can be a numpy array, a list or a numeric value.
+            Numeric values are only valid if the model's output is a rank-n tensor
+            with n <= 2 (regression and classification models).
+            If a numeric value is passed, the gradients are calculated for
+            the same element of the output for all data points.
             For regression models whose output is a scalar, target should not be provided.
             For classification models `target` can be either the true classes or the classes predicted by the model.
+            It must be provided if the model output dimension is higher than 1.
```

Comment:
Make the language consistent to always talk about the rank of the output (instead of dimesnion)?

Reply:
no, but in this case dimension doesn't refer to the rank of the tensor, it refers to the number of classes

Reply:
Ok so in that case the sentence should be changed so it's clear it applies to classification and talks about classes (as opposed to the niche case of 2-class classification with 1-class output probability?).